src/components/options.tsx

```diff
@@ -180,6 +180,10 @@ const JavaScriptPanel = () => {
 	);
 };
 
+const HTMLPanel = () => {
```

Comment:
```suggestion
const HTMLPanel: React.FC = () => {
```

Let's add types for `HTMLPanel` like we used the types in `CssPanel` and `MarkdownPanel`.

![image](https://github.com/user-attachments/assets/156c23c4-3bc1-46b9-b725-84fd0cbdeca4)

![image](https://github.com/user-attachments/assets/3793e2bd-4649-4029-875b-429583528228)